Documentation/Blazorise.Docs/Pages/Blog/2024-07-19_HowToStyleBlazoriseComponents/Index.md

```diff
@@ -0,0 +1,162 @@
+
+Check out the [quick start](https://blazorise.com/docs/start) guide here to get started with Blazorise.
+
+## In this article, we will go over the following:
+
+- Styling Blazorise components#How does CSS work with Blazorise?|How does CSS work with Blazorise?
+- Styling Blazorise components#How to style Blazorise components?|How to style Blazorise components?
+- Styling Blazorise components#Limitations of CSS Isolation|Limitations of CSS Isolation
+
+---
+
+So let's dive into the topics and explore our options when it comes to styling Blazorise components.
+
+## How does CSS work with Blazorise?
+
+Blazorise, just like plain Blazor, is a framework that helps us generate responsive web UI. This means that Blazorise supports every CSS property that is supported by the browser. There are no special CSS properties that only apply to Blazor or Blazorise.
+
+If you would like to find out how to give your Blazorise application a different theme, check out [the docs about theming](https://blazorise.com/docs/theming)
+
+### Inline styling
+
+Inline CSS styling can be applied to any Blazor component directly.
+
+```html|InlineClass
+<div style="color: red;">
+    Hello from Blazorise!
+</div>
+```
+
+Just like regular html elements, Blazor elements can receive all attributes such as style, class, type, and so on.
+
+### CSS classes
+
+Let's take a look at how we can use CSS classes to style our Blazor app.
+
+First we should create a `styles.css` file inside the `wwwroot` folder and add a reference to it inside `App.razor` like so:
+
+create styles.css
+```css|StylesCss
+.bg-topo {
+    background-image: url("...");
+}
+```
+
+App.razor
+```html|AppRazor
+<html>
+<head>
+    <link rel="stylesheet" href="styles.css" />
+</head>
+</html>
+```
+
+After adding this, we can use these classes as usual with our Blazorise components.
+
+Let's see an example:
+```html|UseClassesCss
+<div Class="bg-topo">
+    ...
+</div>
+```
+
+The class will be applied to our div element, and we will see the background image appear.
+
+This is pretty much all the basics covered for regular Blazor
+
+## How to style Blazorise components
+
+Styling Blazorise components is really straightforward. Blazorise being an abstraction over Blazor means that we can directly access the attributes that will be passed to the underlying HTML elements.
+
+### Inline styling
+
+Here is a small example that shows how we can use inline styles with Blazorise.
+
+```html|BlazoriseInlineStylesExample
+<Alert Color="Color.Success" Visible>
+    <AlertMessage>Well done!</AlertMessage>
+    <AlertDescription Style="color:red; font-size:46px">You successfully read this important alert message.</AlertDescription>
+</Alert>
+```
+
+As we can see from [BaseComponent.customStyle](https://github.com/Megabit/Blazorise/blob/master/Source/Blazorise/Base/BaseComponent.cs#L22) all Blazorise components have support for the `Style` and `Class` and many other default attributes.
```

Comment:
Not important to know? Also I follow the link I get to a private field. What do I do with this?